### PR TITLE
Clarify VPC Peering Request Process Statement

### LINKED ISSUE #1307: Clarify customer responsibilities after VPC Peering Request is setup by Astronomer Support
**Affected Document:**
https://docs.astronomer.io/astro/connect-aws?tab=VPC%20peering#connection-options
**Problem Type (Priority):**

- [ X ] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**
This is the problematic paragraph: 
`After receiving your request, Astronomer support initiates a peering request that must be accepted by the owner of the target VPC in your organization. After the VPC peering connection is established, the owner of the target VPC works with Astronomer support to update the routing tables of both VPCs to allow multidirectional traffic.`

I think it's misleading, because it asks the client to update settings in BOTH the Astro VPC and the target VPC, where we only want them responsible for configurations in the target VPC, based on [this slack thread](https://astronomer.slack.com/archives/C01UA03TPV1/p1663013508617739). I think this paragraph should be changed to something like the following: 
`After receiving your request, Astronomer support initiates a peering request that must be accepted by the owner of the target VPC in your organization. Astronomer support will also handle creating route table entries in the Astro VPC. The owner of the target VPC will need to accept the peering request and create route table entries in the target VPC, to allow multidirectional traffic between Airflow and your Data Sources.`

**Additional Notes:**

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1307 

@ben-astro the revised text is ready for your review.

### CODE REVIEW

File: astro/connect-azure.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ To create a VNet peering connection between an Astro VNet and an Azure VNet, con
 - VNet ID of the target VNet
 - Classless Inter-Domain Routing (CIDR) block of the target VNet
 
-After receiving your request, Astronomer support initiates a peering request that must be accepted by the owner of the target VNet in your organization. After the VNet peering connection is established, the owner of the target VNet works with Astronomer support to update the VNet routing tables to allow multidirectional traffic.
+After receiving your request, Astronomer support initiates a peering request and creates the routing table entries in the Astro VPC. To allow multidirectional traffic between Airflow and your organization's data sources, the owner of the target VPC needs to accept the peering request and create the routing table entries in the target VPC.
```
Comment (ben-astro) [inline astro/connect-azure.md:50]: @kwcanuck Can we switch all instances of "VPC" here to "VNet" to more closely align with Azure terminology?

File: astro/connect-aws.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ To create a VPC peering connection between an Astro VPC and an AWS VPC, contact
 - VPC ID of the target VPC
 - Classless Inter-Domain Routing (CIDR) block of the target VPC
 
-After receiving your request, Astronomer support initiates a peering request that must be accepted by the owner of the target VPC in your organization. After the VPC peering connection is established, the owner of the target VPC works with Astronomer support to update the routing tables of both VPCs to allow multidirectional traffic.
+After receiving your request, Astronomer support initiates a peering request and creates the routing table entries in the Astro VPC. To allow multidirectional traffic between Airflow and your organization's data sources, the owner of the target VPC needs to accept the peering request and create the routing table entries in the target VPC.
```
Comment (ben-astro) [inline astro/connect-aws.md:51]: LGTM!

File: astro/connect-gcp.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ To create a VPC peering connection between an Astro VPC and a GCP VPC, contact [
 - VPC ID of the target VPC
 - Classless Inter-Domain Routing (CIDR) block of the target VPC
 
-After receiving your request, Astronomer support initiates a peering request that must be accepted by the owner of the target VPC in your organization. After the VPC peering connection is established, the owner of the target VPC works with Astronomer support to update the routing tables of both VPCs to allow multidirectional traffic.
+After receiving your request, Astronomer support initiates a peering request and creates the routing table entries in the Astro VPC. To allow multidirectional traffic between Airflow and your organization's data sources, the owner of the target VPC needs to accept the peering request and create the routing table entries in the target VPC.
```
Comment (ben-astro) [inline astro/connect-gcp.md:49]: LGTM!

**Review by ben-astro — COMMENTED** [review]
Thanks for addressing! Might need to come back and update Azure doc instructions later, once I confirm details with product team, but this change is helpful as well - especially for the AWS and GCP instructions!

File: astro/connect-azure.md (hunk 1 above)
Comment (kwcanuck) [reply astro/connect-azure.md]: @ben-astro done.

**Review by jwitz — APPROVED** [review]
Looks good!

**Pull request merged**